pkg/controller/atlasinventory/inventory_test.go

```diff
@@ -37,9 +37,9 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
 	dbaasv1alpha1 "github.com/RHEcosystemAppEng/dbaas-operator/api/v1alpha1"
-	dbaas "github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/dbaas"
 	"go.mongodb.org/atlas/mongodbatlas"
 
+	"github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/dbaas"
```

Comment:
are changes to this file necessary?

Reply:
Yes, to pass the lint checking issues.

Reply:
Backed out the change for the formatting issue.